docs/kb/rules/exclude_rule_for_certain_filetypes.md

```diff
@@ -0,0 +1,100 @@
+# How to exclude certain file types for a particular rule
+
+The nature of a certain filetype can generate numerouse false positives and hold up your triage process.  How do you achieve a selective middle ground: continue including the file type to scan against other rules and, at the same time, deflective copious amounts of time triaging false positives?  
+
+## Background
+
+Let's use a real-life case scenario in scanning .svg files as a platform for articulating this solution.   These files mostly comprised a string of thousands of characters:
+
+
+<image id="image0" width="2896" height="998" xlink:href="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAC1AAA**AP**mCAYAAABQS58cAAABR2lDQ1BJQ0MgUHJvZmlsZQAAKJFjYGASSSwoyGFhYGDIzSspCnJ3UoiIjFJgf8LAzsDIwM1gwqCRmFxc4BgQ4ANUwgCjUcG3a0C1QHBZF2SW3AzZBT+7Sn68UphgqTU7fyemehTAlZJanAyk/wBxWnJBUQkDA2MKkK1cXlIAYncA2SJFQEcB2XNA7HQIewOInQRhHwGrCQlyBrJvANkCyRmJQDMYXwDZOklI4ulIbKi9IMDj4urjoxBqZG5oEUjAuaSDktSKEhDtnF9QWZSZnlGi4AgMpVQFz7xkPR0FIwMjAwYGUJhDVH8OAoclo9g+hFj+EgYGi28MDMwTEWJJUxgYtrcxMEjcQ…..
+
+
+
+Semgrep’s standard artifactory rule, for example, reports on:   
+
+_\# ruleid: detected-artifactory-password_
+
+_AP6xxxxxxxxxx_
+
+_\# ruleid: detected-artifactory-password_
+
+_AP2xxxxxxxxxx_
+
+...
+
+
+We can quickly see how the match in bold above will flag the artifactory rule as a false positive.  
+
+
+## Choosing the appropriate muting solution
+
+
+Semgrep offers many different ways of muting false positives: 
+
+One could employ the nosemgrep directive but this would require having to keep track of each new file for this target .svg file type and editing it accordingly, creating a layer of maintenance with this muting notation. 
+
+One could ignore the file entirely, placing it in the .semgrepignore listing, but then that would preclude this type of file from scanning against not only the artifactory rule, but all rules.  
+
+
+## Achieving a happy medium
+
+
+How can we achieve a happy medium of selectively precluding this filetype from the artifactory rule, but still scan .svg files against all other rules?
+
+We can safely assume .svg files do not intentional contain artifactory passwords, so we are safe to preclude this rule from scanning against this filetype and the following procedure demonstrates how to:
```

Comment:
```suggestion
You can safely assume `.svg` files do not intentionally contain artifactory passwords, so you can exclude scanning against this file type. The following procedure demonstrates how to:
```

You can address the audience as "you" ([Google Style Guide - Second and First person](https://developers.google.com/style/person#:~:text=your%20audience%20consistently-,Address%20the%20reader%20as%20you,tasks%20that%20you're%20documenting.))